run/events-pubsub/app.js

```diff
@@ -20,9 +21,10 @@ app.post('/', (req, res) => {
     console.log(`Bad Request: ${errorMessage}`);
     return;
   }
-  const pubSubMessage = req.body.message;
-  const name = pubSubMessage.data
-    ? Buffer.from(pubSubMessage.data, 'base64').toString().trim()
+  // Cast to MessagePublishedEvent for IDE autocompletion
+  const pubSubMessage = toMessagePublishedEvent(req.body);
```

Comment:
Nit: would `message` be too non-descript here? (It's shorter 🙂 )

Reply:
It would be non-descript.

I'm not going to write `data.message.data`.

_(Flashbacks to [`data.data.data`](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/1474))_